src/lib/utils/renderMarkdown.tsx

```diff
@@ -0,0 +1,165 @@
+      )
+    },
+  },
+
+  text: {
+    ...SimpleMarkdown.defaultRules.text,
+    text: {
+      react: node => {
+        return node.content
+      },
+    },
+  },
+
+  paragraph: {
+    ...SimpleMarkdown.defaultRules.paragraph,
+    match: SimpleMarkdown.blockRegex(/^((?:[^\n]|\n(?! *\n))+)(?:\n *)/),
+    react: (node, output, state) => {
+      return (
+        <Sans size="3t" color="black60" key={state.key} textAlign="center">
+          {output(node.content, state)}
+        </Sans>
+      )
+    },
+  },
+
+  strong: {
+    ...SimpleMarkdown.defaultRules.strong,
+    react: (node, output, state) => {
+      return (
+        <Serif size="3t" weight="semibold" key={state.key}>
+          {output(node.content, state)}
+        </Serif>
+      )
+    },
+  },
+
+  em: {
+    ...SimpleMarkdown.defaultRules.em,
+    react: (node, output, state) => {
+      return (
+        <Serif size="3t" italic key={state.key}>
+          {output(node.content, state)}
+        </Serif>
+      )
+    },
+  },
+
+  br: {
+    ...SimpleMarkdown.defaultRules.br,
+    react: (_node, _output, state) => {
+      return <Text key={state.key} />
+    },
+  },
+
+  newline: {
+    ...SimpleMarkdown.defaultRules.newline,
+    react: (_node, _output, state) => {
+      return <Text key={state.key} />
+    },
+  },
+
+  list: {
+    ...SimpleMarkdown.defaultRules.list,
+
+    react: (node, output, state) => {
+      const items = _.map(node.items, (item, i) => {
+        let bullet
+        if (node.ordered) {
+          bullet = <Serif size="3t" key={state.key}>{`${i + 1} . `}</Serif>
+        } else {
+          bullet = (
+            <Serif size="3t" key={state.key}>
+              -{" "}
+            </Serif>
+          )
+        }
+
+        const listItemText = (
+          <Serif size="3t" key={state.key + 1}>
+            {output(item, state)}
+          </Serif>
+        )
+        return (
+          <View key={i} style={{ flexDirection: "row" }}>
+            {bullet} {listItemText}
+          </View>
+        )
+      })
+      return <View>{items}</View>
+    },
+  },
+
+  codeBlock: {
+    react: (node, output, state) => {
+      return (
+        <Sans size="3t" key={state.key}>
+          {output(node.content, state)}
+        </Sans>
+      )
+    },
+  },
+
+  inlineCode: {
+    react: (node, output, state) => {
+      return (
+        <Sans size="3t" key={state.key}>
+          {output(node.content, state)}
+        </Sans>
+      )
+    },
+  },
+
+  heading: {
+    ...SimpleMarkdown.defaultRules.heading,
+    react: (node, output, state) => {
+      const map = {
+        1: "8",
+        2: "6",
+        3: "5t",
+        4: "5",
+      }
+      const size = map[node.level] || "4"
+      return (
+        <Sans mb="1" key={state.key} size={size}>
+          {output(node.content, state)}
+        </Sans>
+      )
+    },
+  },
+}
+
+export function renderMarkdown(markdown: string, rules: any = defaultRules): React.ReactElement {
```

Comment:
Do you think we should add some unit tests for this?

Reply:
They probably already exist for the `Markdown` component but I'll add some here.